lnurl-withdraw.md

```diff
@@ -17,8 +17,8 @@ Today users are asked to provide a withdrawal Lightning invoice to a service, th
         callback: String, // the URL which LN SERVICE would accept a withdrawal Lightning invoice as query parameter
         k1: String, // random or non-random string to identify the user's LN WALLET when using the callback URL
         defaultDescription: String, // A default withdrawal invoice description
-        maxWithdrawable: MilliSatoshi, // max withdrawable amount for a given user on LN SERVICE
-        minWithdrawable: MilliSatoshi, // An optional field, defaults to 1 MilliSatoshi if not present, can not be less than 1 or more than `maxWithdrawable`
+        minWithdrawable: Integer, // Min amount (in millisatoshis) the user can withdraw from LN SERVICE, or 0
```

Comment:
Nit: "Min amount or 0" sounds a bit funny to me, I'd word it as "Min amount, can be 0"